spec/models/default_bill_spec.rb

```diff
@@ -5,17 +5,21 @@
 RSpec.describe DefaultBill do
   let!(:current_user) { create(:user) }
 
-  it 'is valid with valid params' do
-    expect(described_class.new(name: 'Default Bill', user: current_user)).to be_valid
-  end
+  context 'with valid params' do
+    it 'is valid' do
+      default_bill = described_class.new(name: 'Default Bill', user: current_user)
+
+      expect(default_bill).to be_valid
+    end
 
-  it 'generates slug' do
-    default_bill = described_class.create(name: 'Default Bill', user: current_user)
+    it 'generates slug' do
+      default_bill = described_class.create(name: 'Default Bill', user: current_user)
 
-    expect(default_bill.slug).to match(/default-bill-.*/)
+      expect(default_bill.slug).to match('default-bill')
+    end
   end
 
-  describe 'model validations' do
+  describe '#model validations' do
```

Comment:
```suggestion
  describe 'model validations' do
```

esse `#` a gente põe quando é referente a um método de instância

por exemplo, vamos supor que vc criou um método novo dentro do model DefaultBill, chamado `blah`, daí aqui no teste vc pode fazer um `describe "#blah" do`

e quando é um método de classe, aí a gente põe um `.` na frente

Reply:
No máximo eu acharia aceitável um `describe "#valid?" do`

Reply:
Entendi 🤔 
Então qual seria a melhor maneira de nomear isso? Porque se não me engano, quando você usa um `describe` ele não dá um espaço no nome do teste, não?
Por exemplo, 
`describe 'model validations'` seria um: `DefaultBillmodel validations it validates name presence`, ou estou errado?

Reply:
É, na verdade eu to errado 😂 
Quando tem o `#`, ele não dá o espaço: 
![image](https://user-images.githubusercontent.com/48293264/212393590-5256c22b-d646-4611-be1f-b7c1df87979d.png)

Mas quando não tem o `#`, ele dá o espaço:
![image](https://user-images.githubusercontent.com/48293264/212393651-004136b5-294e-4073-844f-a0dbb25d60fb.png)